.kokoro/nightly/retry_non_quota.cfg

```diff
@@ -0,0 +1,11 @@
+# Format: //devtools/kokoro/config/proto/build.proto
```

Comment:
Also add some other env vars that the kokoro set up require.
[This](https://github.com/googleapis/java-bigquery/blob/main/.kokoro/nightly/nightly-integration.cfg) file is a good reference.

Reply:
Done; let me know if I missed something.  Additionally, is the test runner's SA java-it-service-account or it-service-account?  Currently only it-service-account@gcloud-devel.iam.gserviceaccount.com is allowlisted for the special projects.

Reply:
I believe, java-it-service-account is the name for the secret manager where the key is stored.